packages/eslint/eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import { eslint } from './src/index.mjs';
+
+export default [
```

Comment:
I would add a comment explaining that this config is just for this package, not to be confused with `index.mjs` which is the export